server/flaskr/__init__.py

```
@@ -5,21 +5,20 @@
 from .database import init_db
 from .blueprints import register_blueprints
 
-def create_app():
+def create_app(config_name='development'):
     load_dotenv()
 
-    app = Flask(__name__, instance_relative_config=True)
+    app = Flask(__name__)
+
     app.config.from_mapping(
         SECRET_KEY='dev',
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to argument: "SECRET_KEY"

(S106)